upgrade_test.py

```diff
@@ -23,16 +23,17 @@
 from functools import wraps, cache
 from typing import List
 
-from argus.db.db_types import PackageVersion
 from pkg_resources import parse_version
+from argus.db.db_types import PackageVersion
 
 from sdcm import wait
 from sdcm.cluster import BaseNode
 from sdcm.fill_db_data import FillDatabaseData
 from sdcm.stress_thread import CassandraStressThread
 from sdcm.utils.version_utils import is_enterprise, get_node_supported_sstable_versions
 from sdcm.sct_events.system import InfoEvent
-from sdcm.sct_events.database import IndexSpecialColumnErrorEvent
+from sdcm.sct_events.filters import DbEventsFilter
```

Comment:
Add this after line 37.

All these playings with imports are messing the backports afterwards.